charts/federation-v2/templates/statefulset.yaml

```diff
@@ -28,6 +28,18 @@ spec:
       containers:
       - args:
         - --install-crds=false
+{{- if .Values.controllermanager.featureGates }}
+        - --feature-gates={{ .Values.controllermanager.featureGates }}
+{{- end }}
+{{- if .Values.controllermanager.fedNamespace }}
+        - --federation-namespace={{ .Values.controllermanager.fedNamespace }}
+{{- end }}
+{{- if .Values.controllermanager.clusterNamespace }}
+        - --registry-namespace={{ .Values.controllermanager.clusterNamespace }}
+{{- end }}
+{{- if .Values.controllermanager.limitedScope }}
```

Comment:
I think enabling limited scope implies setting the federation and registry namespaces to the namespace the controller manager is deployed in (e.g. [1]).  Also, if limitedScope is true, the service account that the controller manager runs under should be restricted to the federation namespace.


1: https://github.com/kubernetes-sigs/federation-v2/blob/master/scripts/generate-namespaced-install-yaml.sh#L39

Reply:
I see, let me update.

Reply:
Done @marun